onyo/lib/recorders.py

```diff
@@ -28,14 +30,22 @@ def record_item(repo: OnyoRepo, item: Path | dict) -> str:
 
 
 def record_move(repo: OnyoRepo, src: Path | dict, dst: Path) -> str:
-    # Attention: This currently expects `dst` to be the dir to move src into,
-    # rather than already containing src' name at the destination. This may not be consistent yet.
+    # This currently expects `dst` to be the dir to move src into,
+    # rather than already containing src' name at the destination.
     src_path = src if isinstance(src, Path) else src['path']
     dst_path = (dst / src_path.name).relative_to(repo.git.root).as_posix()
     src_path = src_path.relative_to(repo.git.root).as_posix()
     return f"- {src_path} -> {dst_path}{linesep}"
 
 
+def record_rename(repo: OnyoRepo, src: Path | dict, dst: Path) -> str:
```

Comment:
This seems to be a whole new function, maybe here a docstring, test, and/or the linebreaks for multiple arguments for readability would already make sense (I find it hard to quickly scan especially when having e.g. unions like `Path | dict`), more so because it is meant as a bug fix :)

Reply:
I'd agree in general.
However, in case of executors, recorders and differs, nothing has proper docstrings yet, they all have largely identical signatures, are very simple code-wise and none of that is intended for any kind of third-party usage.
That's why I didn't bother yet, apart from an introductory comment at the top of each of those three files.

Reply:
Yes, thanks a lot for the introductory comments! :)